lib/presentation/features/details/bloc/details_bloc.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvelapp_flutter/data/repository/marvell_repository.dart';
+import 'package:marvelapp_flutter/presentation/features/details/bloc/details_event.dart';
+import 'package:marvelapp_flutter/presentation/features/details/bloc/details_state.dart';
+
+class DetailsBloc extends Bloc<DetailsEvent, DetailsState> {
+  final MarvellRepository repository;
+  final String characterId;
```

Comment:
We should have `characterId` parameter here. it should be passed via `GetCharacterDetail`:
```
class GetCharacterDetail extends DetailsEvent {
  final String characterId;
```

Reply:
done